### PR TITLE
add GH Actions workflow for running tests

### PR DESCRIPTION
**Pull request by cmccandless**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+jobs:
+  test:
+    name: Test Runner
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Build Docker Image
+        run: docker build -f Dockerfile -t c-test-runner .
```
Comment (bergjohan) [inline .github/workflows/main.yml:24]: Could you change the tag name from c-test-runner to x86-64-assembly-test-runner?

Comment (cmccandless) [reply .github/workflows/main.yml:24]: Oops; missed that. Addressing that now.

**Comment by bergjohan** [comment]
Merged! :)

**Comment by bergjohan** [comment]
And it works like a charm: https://github.com/exercism/x86-64-assembly-test-runner/runs/832385042

**Pull request merged**